packages/v4/src/content/community/extensions-data.js

```diff
@@ -0,0 +1,58 @@
+export const communityExtensions = [
+  {
+    "component": "Catalog View",
+    "description": "Implements a catalog view using PatternFly components",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/catalog-view-item-header"
+      },
+      {
+        "name": "Code",
+        "href": "https://www.npmjs.com/package/@patternfly/react-catalog-view-extension"
+      }
+    ]
+  },
+  {
+    "component": "Console",
+    "description": "React-console extension",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/access-consoles"
```

Comment:
Small copy edit for the access console extension example: Let's rewrite the example title "Basic Usage" in sentence case: _Basic usage_.